.github/scripts/generate_binary_build_matrix.py

```diff
@@ -58,6 +58,21 @@
         "nvidia-nccl-cu12==2.20.5; platform_system == 'Linux' and platform_machine == 'x86_64' | "
         "nvidia-nvtx-cu12==12.1.105; platform_system == 'Linux' and platform_machine == 'x86_64'"
     ),
+    "12.4": (
+        "nvidia-cuda-nvrtc-cu12==12.3.107; platform_system == 'Linux' and platform_machine == 'x86_64' | " 
+        "nvidia-cuda-runtime-cu12==12.4.99; platform_system == 'Linux' and platform_machine == 'x86_64' | "
+        "nvidia-cuda-cupti-cu12==12.4.99; platform_system == 'Linux' and platform_machine == 'x86_64' | "
+        "nvidia-cudnn-cu12==9.0.0.322; platform_system == 'Linux' and platform_machine == 'x86_64' | "
```

Comment:
`nvidia-cudnn-cu12==9.0.0.322` does not exist, see: https://github.com/pytorch/pytorch/pull/120642#issuecomment-1974017028

Reply:
Using 8.9.7.29 same as what is used in https://github.com/pytorch/pytorch/pull/120642 thanks for the pointer!